--- mbed/TARGET_LPC1768/TOOLCHAIN_GCC_CR/LPC1768.ld	Fri Apr 17 16:02:21 2015
+++ mbed/TARGET_LPC1768/TOOLCHAIN_GCC_CR/LPC1768.ld	Fri Apr 10 15:35:08 2015
@@ -134,6 +134,9 @@
     {
         *(.bss.$RAM2*)
         *(.bss.$RamAHB32*)
+        *FATFileSystem*(.bss*)
+        *EthernetInterface*(.bss*)
+        *EthernetInterface*(COMMON)
        . = ALIGN(4) ;
     } > RamAHB32
 
@@ -151,4 +154,14 @@
     
     PROVIDE(_pvHeapStart = .);
     PROVIDE(_vStackTop = __top_RamLoc32 - 0);
+    
+    AHBSRAM0 (NOLOAD):
+    {
+        *(AHBSRAM0)
+    } > RamAHB32
+    
+    AHBSRAM1 (NOLOAD):
+    {
+        *(AHBSRAM1)
+    } > RamAHB32
 }
